{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--4b334a3e-1557-40ca-aaa8-71c6f9e5db46",
    "created": "2024-08-13T16:13:31.06305Z",
    "modified": "2024-08-13T16:13:31.063054Z",
    "name": "No Title Available",
    "description": "api/views/user.py in LibrePhotos before e19e539 has incorrect access control.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2023-22903"
        }
    ],
    "references": [
        {
            "url": "https://github.com/LibrePhotos/librephotos/commit/e19e539356df77f6f59e7d1eea22d452b268e120"
        },
        {
            "url": "https://raw.githubusercontent.com/go-compile/security-advisories/master/CVE-2023-22903.pdf"
        }
    ]
}